vsphere/image/settings.auto.pkrvars.hcl

```diff
@@ -0,0 +1,33 @@
+/*
+    DESCRIPTION:
```

Comment:
can some of the values be moved to `variables.pkr.hcl` as default values and this file be removed ?

Reply:
The way I envisioned the design for this is that the *auto*.hcl files contain the user facing name=value settings that are most likely to change for a custom template build. Ideally, I would think that users should not have to alter the variables file to edit default values which has a more "structured" syntax and IMO is easier to break. The examples I followed from https://github.com/vmware-samples/packer-examples-for-vsphere also seem to follow this approach. Do you see any disadvantages to this ?

Reply:
Fair point. I was going by the pattern where any custom values will be passed as env variables and handled in the Makefile target. Example - https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/aws/image/Makefile#L73
If having the *.auto.pkrvars.hcl files then we need to decide whether to track it or add it to `.gitignore`. 

Another aspect to consider is the number of input params required. For a large number of params `*.auto.pkrvars.hcl` seems more convenient. Hence I was thinking if some of the defaults are common then those can be added to `variables.pkr.hcl` 
Anyways, I'll leave the decision to you. Both approaches are fine :-)

Reply:
Thanks @bpradipt I pushed new changes and updated the cover letter to summarize them.